src/main/steps/c100-rebuild/c100sequence.test.ts

```diff
@@ -4,7 +4,7 @@ import { C100Sequence } from './c100sequence';
 
 describe('C100Sequence', () => {
   test('should contain 1 entries in c100 screen sequence', () => {
-    expect(C100Sequence).toHaveLength(9);
+    expect(C100Sequence).toHaveLength(10);
```

Comment:
@rahulsanjava19 Can you please add test case for new URL ?
`/reasonable-adjustments/attending-court`

Reply:
done